draft-ietf-ppm-dap.md

```diff
@@ -152,6 +152,16 @@ input is ever seen in the clear by any aggregator.
 
 12:
 
+- Bump draft-irtf-cfrg-vdaf-08 to 12 {{!VDAF}}. (\*)
+
+- Add support for "asynchronous" aggregation, based on the Leader polling the
+  Helper for completion. (\*)
+
+- Update collection semantics to match the new aggregation semantics introduced
```

Comment:
Is it the semantics that we're updating or just the "syntax"?

Reply:
I think I prefer "semantics" -- e.g. we're now indicating the current status of a collection job by an in-message status field, rather than using HTTP status. IMO, that's a semantic change.